src/adapter.c

```diff
@@ -261,6 +261,7 @@ int init_hw(int i)
 	ad->fe_sock = -1;
 	ad->sock = -1;
 	ad->force_close = 0;
+	ad->is_closing = 0;
```

Comment:
Move this to satipc.c

Reply:
> Move this to satipc.c

Sorry? `is_closing` is an adapter member and requires to be INITIALIZED!! Here is the initialization of the adapter. Why you say to set _only_ in the `satipc` module?